src/pages/HomePage/HomePage.jsx

```diff
@@ -0,0 +1,58 @@
+import { Box } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import { createContext, useEffect, useState } from 'react';
+
+import Footer from '../../components/Footer/Footer';
+import Header from '../../components/Header/Header';
+// eslint-disable-next-line import/no-cycle
```

Comment:
If you leave the context it would be a good idea to move it into a separate file to avoid circular imports.